ci/docker/connector_a_resources/application-docker.properties

```diff
@@ -0,0 +1,65 @@
+spring.application.name=connector
+application.automatic.negotiation=false
+
+server.port=8080
+application.callback.address=${CALLBACK_ADDRESS}
+
+spring.application.label= latest
+
+# MongoDB connection settings
+#spring.data.mongodb.authentication-database=admin
+spring.data.mongodb.host=mongodb-a
+spring.data.mongodb.port=27017
+spring.data.mongodb.database=true_connector
+#spring.data.mongodb.username=tc
+#spring.data.mongodb.password=password
+
+####
+logging.level.org.springframework=INFO
+
+spring.ssl.bundle.jks.connector.key.alias=${KEYSTORE_ALIAS}
+spring.ssl.bundle.jks.connector.key.password=${KEY_PASSWORD}
+spring.ssl.bundle.jks.connector.keystore.location=/cert/${KEYSTORE_NAME}
+spring.ssl.bundle.jks.connector.keystore.password=${KEYSTORE_PASSWORD}
+#add private key password
+spring.ssl.bundle.jks.connector.keystore.type=JKS
+spring.ssl.bundle.jks.connector.truststore.type=JKS
+spring.ssl.bundle.jks.connector.truststore.location=/cert/${TRUSTSTORE_NAME}
+spring.ssl.bundle.jks.connector.truststore.password=${TRUSTSTORE_PASSWORD}
+
+management.endpoints.enabled-by-default=true
+management.endpoint.info.enabled=true
+#management.security.enabled=false
+management.endpoints.web.exposure.include=*
+management.endpoint.env.post.enabled=true 
+
+### Connector ID
+
+application.connectorid=29d022fa-33be-4627-923e-412eb609eb6b
```

Comment:
Should we put some "meaningful" value for ID?
Even for testing

Reply:
What did you have in mind? Something like connector_a, connector_b?

Reply:
We could start with those values and see how will that one go. Depends where is this value used

Reply:
I have changed it